packages/flame/lib/src/game/base_game.dart

```diff
@@ -20,13 +19,10 @@ import 'viewport.dart';
 /// This is a more complete and opinionated implementation of Game.
 ///
 /// BaseGame should be extended to add your game logic.
-/// [update], [render] and [onResize] methods have default implementations.
+/// [update], [render] and [onGameResize] methods have default implementations.
 /// This is the recommended structure to use for most games.
 /// It is based on the Component system.
 class BaseGame extends Game {
```

Comment:
Would it make sense to call this `FCSGame`, or `ComponentBasedGame`, or something like that, in order to clearly distinguish it from `OxygenGame` and `Forge2DGame`?

Reply:
I like `BaseGame` as a name, since it is the base game in Flame and since `OxygenGame` and `Forge2DGame` lives outside of Flame. `Forge2DGame` even extends `BaseGame`. 
I think adding `FCS` there instead will confuse users more than helping them.  
(I don't like acronyms in names either)

Reply:
Could be that I'm just very used to it though, we could have a poll for this in discord among the users if any more people of the team think that it could be a good idea to change the name. :)

Reply:
Yeah, i don't like 'FCS' either. How about `FlameGame`? (I don't know how to do polls on discord)

Reply:
What is wrong with `BaseGame` though? I'm open for a discussion to change it, but I don't see any problems with the current name.

Reply:
Well, I'm just trying to think which name would best capture the intent and purpose of that class.
For example, the `Game` class can be described as "The base game class", because it's at the root of the hierarchy of various game classes. The `BaseGame` class is currently described as a "more opinionated implementation of a Game", but that's not very precise. We could say that `BaseGame` is a "component-based implementation of a Game", so it's a base class for all Flame-engine games, but not **the** base game class. So, a `FlameGame` could be appropriate, but only if my understanding of the scope of the name "Flame" is correct.

Reply:
originally `more opinionated implementation` meant not only adding components, but a lot of other "default" things. I am not sure if that is the case anymore, but that was the original idea and hence the name

Reply:
Opened discussion #948, since this can be done as a follow-up after this PR is merged.